test/valid-packages-test.js

```diff
@@ -306,6 +306,22 @@ packages.map(function(pkg) {
                 pkgName(pkg) + ": Need to remove \"/\" at the front or the last of basePath in package.json");
         }
       }
+  }
+  packageVows[pname + ": There must not have duplicate keywords"] = function(pkg) {
+      var json = parse(pkg, true);
+      var kw = json.keywords;
+      var dul_key = [];
+      if (kw != undefined) {
+        kw.sort();
+        dul_key = kw.filter (function (dul, i, temp) {
+            if (i >= 0 && dul == temp[i-1]) {
+              return dul;
+            }
+        });
+      }
+      assert.ok(
+          dul_key.length <= 0,
+          pkgName(pkg) + ": Need to remove duplicate keywords: " + dul_key);
```

Comment:
@pvnr0082t 
It will show the error `SyntaxError: Unexpected token =`
So I think we can't use this.